src/Sinking.Web/Components/Pages/Home.razor.css

```
@@ -0,0 +1,386 @@
+    margin: 0;
+}
+
+.activity-card {
+    background: var(--dark-surface);
+    border: 1px solid var(--border-color);
+    border-radius: var(--radius-lg);
+    overflow: hidden;
+}
+
+.activity-list {
+    divide-y: 1px solid var(--border-color);
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)